src/Components/Publishing/Header/BasicHeader.tsx

```diff
@@ -208,41 +213,32 @@ const Container = div`
 `
 
 const Vertical = styled.div`
-  ${defaults}
-
-  ${breakpoint.xs`
+  ${defaults} ${breakpoint.xs`
     ${Fonts.unica("s14", "medium")}
-  `}
+  `};
 `
 
 const Title = styled.div`
-  ${Fonts.unica("s80", "regular")}
-  line-height: 1.1em;
+  ${Fonts.unica("s80", "regular")} line-height: 1.1em;
   letter-spacing: -0.035em;
   max-width: 1250px;
   margin: 0 auto 27px auto;
   ${breakpoint.md`
     font-size: 60px;
-  `}
-
-  ${breakpoint.xs`
+  `} ${breakpoint.xs`
     font-size: 40px;
     margin-bottom: 15px;
-  `}
+  `};
 `
 
 const LeadParagraph = styled.div`
-  ${defaults}
-
-  ${breakpoint.xs`
+  ${defaults} ${breakpoint.xs`
```

Comment:
Ah yeah Prettier does this but this can be separated by line by adding a semicolon. Aka this wouldn't condense those lines:
```
${defaults};
${breakpoint.xs``}
```

Reply:
Don't feel like you have to go through and do those but just to note!